Frontend/ui-customer/src/Flow.purs

```diff
@@ -2306,10 +2320,38 @@ zooTicketBookingFlow = do
       setValueToLocalStore PAYMENT_STATUS_POOLING "false"
       zooTicketBookingFlow
 
+dummyServiceDetailsNON :: ST.TicketBookingServiceDetails
+dummyServiceDetailsNON = 
+  { amount : 55.0,
+    status : "dkfj;a",
+    verificationCount : 5,
+    expiryDate : Just "akfjk;asd",
+    ticketServiceName : "dkj;akl",
+    categories : [dummyCategoryNON, dummyCategoryNON],
+    ticketServiceShortId : "dkajf;a",
+    slot : Just "13:00:00"
+  }
+
+dummyCategoryNON :: ST.TicketBookingCategoryDetails
```

Comment:
Here also 
Move these dummies in screen data or somewhere else

Reply:
done